pkgs/stdenv/generic/check-meta.nix

```diff
@@ -86,12 +89,41 @@ let
     allowInsecurePredicate attrs ||
     builtins.getEnv "NIXPKGS_ALLOW_INSECURE" == "1";
 
-  showLicense = license: toString (map (l: l.shortName or "unknown") (lib.lists.toList license));
+  hasSourceProvenance = attrs:
```

Comment:
The name `hasSourceProvenance` is sort of ambiguous: "has `meta.sourceProvenance`" vs "has source-code provenance".

Since `hasSourceProvenance` is used in only one place, why not just inline it?